doc/gov/chapters/chapter09.md

```diff
@@ -0,0 +1,83 @@
+
+<a name="9.2"></a>
+## 9.2 Transition Plan
+
+A Transition plan comprised of one or more exceptions and/or transitions is required to address technology that does not presently conform to CNTT mandates, and hence requires explicit direction to prescribe how the situation will be treated in the present, as well as in the future.
+
+The transition plan described here will inform application designers how RC and ultimately industry certification programs will react when encountering exceptions during the qualification process, including flagging warnings and potentially errors which could prevent issuance of a certification badge.
+
+<a name="9.2.1"></a>
+### 9.2.1 Conformance Levels
+
+- **Fully Conformant**: VNFs/CNFs or NFVI are written and designed to be fully conformant to CNTT specification with no use of any of the allowed Exceptions.
+- **Conformant with Exceptions**: VNFs/CNFs or NFVI are written and designed to be conformant to CNTT with one or more of the allowed Exceptions used.
+
+<a name="9.2.2"></a>
+### 9.2.2 Exception Types
+
+- **Technology Exceptions** : Using specific technologies that are considered non conformant to CNTT principles (such as PCIe Direct Assignment, exposure of hardware features to VNFs/CNFs).
+- **Version Exceptions**: Using Versions of  Software components, , APIs, or Hardware that are different from the one specified in the specification.
+
+<a name="9.2.3"></a>
+### 9.2.3 Transition Framework
+
+#### VNF/CNF Transition Plan Framework 
+
+Exceptions will be clearly recorded in the Reference Model Appendix which will act as a guidance to VNFs/CNFs vendors of what Exceptions will be allowed in each CNTT release. **Figure 1** below demonstrate the concept.
+
+- As technology matures, fewer and fewer Exceptions will be allowed in CNTT releases.
+- For each CNTT Release, VNF/CNF can be either:
+  - **Fully Conformant**: No Exception used.
+  - **Conformant with Exception**: One or More of the allowed Exceptions in RM has been used.  
+
+
+<p align="center"><img src="../figures/vnf_cnf_transition.png" alt="Transition" title="VNF/CNF Transition Plan" width="70%"/></p>
+<p align="center"><b>Figure 1:</b> Transition Plan for VNFs/CNFs within CNTT</p>
+
+#### NFVI Transition Plan Framework 
+
+Exceptions will be clearly recorded in Reference Architectures' Appendices which will act as a guidance to NFVI vendors of what Exceptions will be allowed in each CNTT release. **Figure 2** below demonstrate the concept.
+
+- As technology matures, fewer and fewer Exceptions will be allowed in CNTT releases.
+- For each CNTT Release, VNF/CNF can be either:
+  - **Fully Conformant**: Support Target Architecture with all RM Exceptions supported. There should be a technology choice in RA to support RM Exceptions (However, none of the Exceptions allowed in RA has been used).
```

Comment:
Suggest change "Support Target Architecture with all RM Exceptions supported" to "Support the Target Reference Architecture without any exceptions."

Reply:
fixed in here: https://github.com/cntt-n/CNTT/pull/893/commits/74786114e8ed0a1a3085d00a010e55089540381e